UI/ChatWnd.cpp

```diff
@@ -441,6 +441,8 @@ void MessageWnd::HandleTurnPhaseUpdate(Message::TurnProgressPhase phase_id) {
         break;
     }
 
+    if (prefixed)
+        *m_display += UserString("PLAYING_GAME") + " ";
```

Comment:
prefer a separate string for this, with `boost::format` for substitution, to explicit string building in C++ code

Reply:
Fixed